src/R/Components/Impl/ConnectionManager/Implementation/ConnectionManager.cs

```diff
@@ -64,6 +64,12 @@ internal class ConnectionManager : IConnectionManager {
             _userConnections = new ConcurrentDictionary<Uri, IConnection>(userConnections);
 
             UpdateRecentConnections();
+
+            _shell.Idle += OnIdle;
+        }
+
+        private void OnIdle(object sender, EventArgs e) {
```

Comment:
This may not be enough. It is possible that root controls aren't added yet, so TryAddItemsControlToVisualRoot won't find a parent panel.

---
In reply to: [78587125](https://github.com/Microsoft/RTVS/pull/2296#discussion_r78587125) [](ancestors = 78587125)